jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,456 @@
+    "<br>\n",
+    "<hr>\n",
+    "\n",
+    "## Prerequisites\n",
+    "Before you run this example, you need to [create a DataFinder search query](https://www.ni.com/documentation/en/systemlink/latest/datanavigation/finding-data-with-advanced-search/) in Data Navigation to find the example files (e.g. 'TR_M17_QT_42-1.tdms'). Save this query on the server.\n",
+    "<hr>\n",
+    "\n",
+    "## Summary\n",
+    "\n",
+    "This example exercises the SystemLink TDMReader API to access bulk data (see `data_api`) and/or descriptive data (see `metadata_api`). When the notebook executes, Analysis Automation provides data links which the API uses to access content.\n",
+    "\n",
+    "It also shows how to select channels from two channel groups and display the data in two graphs.\n",
+    "\n",
+    "The channel values from each channel group populate arrays, which you can use to further analyze your data in mathematical analyses and plot them in a graph.\n",
+    "\n",
+    "Furthermore, the example uses two procedure parameters that write a comment to the first graph and select a channel to display in the second graph (refer to __Plot Graph__ below).\n",
+    "\n",
+    "\n",
+    "\n",
+    "<hr>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports\n",
+    "\n",
+    "This example uses the `TDMReader` API to work with the bulk data and meta data of the given files. `Matplotlib` is used for plotting the graph. The `scrapbook` is used to set and display the results in the analysis procedure results list. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import systemlink.clients.nitdmreader as tdmreader\n",
+    "metadata_api = tdmreader.MetadataApi()\n",
+    "data_api = tdmreader.DataApi()\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import scrapbook as sb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_property(element, property_name):\n",
+    "    \"\"\"Gets a property of the given element.\n",
+    "    \n",
+    "    The element can be a file, channel group, or channel.\n",
+    "    \n",
+    "    Args:\n",
+    "        element: Element to get the property from.\n",
+    "        property_name: Name of the property to get.\n",
+    "        \n",
+    "    Returns:\n",
+    "        The according property of the element or ``None`` if the property doesn't exist.\n",
```

Comment:
"according" here seems like the wrong word to use.  Perhaps "respective" or simply leave the word out.

Reply:
Thanks. Took it out.